### ISSUE TITLE
Markdown plugin: add markdown slides at runtime

### ISSUE THREAD

**Issue by croeck** [hakimel/reveal.js#586]
Hi,

I would love to add slides to my presentation while it is already running. With most aspects of reveal.js this is working great. However, the markdown slides are parsed while reveal.js is being initialized and currently there is no possibility to re-run the markdown plugin.

First, I would request to provide a simple API call for the markdown plugin to initialize all markdown slides, similar to reveal.js itself.

The following code refers to the bottom of the `markdown.js` file in the markdown plugin and would implement this functionality:

``` javascript
    var initialize = function() {
        querySlidingMarkdown();
        queryMarkdownSlides();
    };

    querySlidingMarkdown();
    queryMarkdownSlides();

    // --------------------------------------------------------------------//
    // ------------------------------- API --------------------------------//
    // --------------------------------------------------------------------//

    return {
        initialize: initialize
    };
```

Second, this would require to change the existing methods because already parsed slides would be parsed again and therefore become corrupted.
My first thought was to remove the `data-markdown` attribute after a section has been parsed and add a `data-markdown-parsed` attribute instead.

Basically both changes are working as desired, but I am not quite sure about the side-effects this would have, especially on the notes part of reveal.js

I would love to hear your comments on this.

(And thanks for the already fantastic tool!)
